### PR TITLE
Improve character title

### PR DESCRIPTION
**Pull request by hskim881028**
### Description

- Detach character title from the HP bar. Add HudContainer. Then load the HP bar and title into the container.
- Improve character title 
- Add Resource Helper. It manages scriptable objects.

### How to test

- Equip character title

### Related Links

- [Link](https://app.asana.com/0/958521740385861/1198932974018853/f)

### Required Reviewers

@planetarium/9c-dev , @Namyujeong 

### Screenshot

image

### CODE REVIEW

File: nekoyume/Assets/_Scripts/Helper/ResourcesHelper.cs [hunk 1]
```diff
@@ -0,0 +1,51 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using TMPro;
+using UnityEngine;
+
+namespace Nekoyume.Helper
+{
+    public static class ResourcesHelper
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/Helper/ResourcesHelper.cs:8]: How about this initializing asynchronously like below.
```cs
// in Game.cs
await ResourceHelper.InitializeAsync();
or
yield return StartCoroutine(ResourceHelper.CoInitialize());
```

**Review by Namyujeong — CHANGES_REQUESTED** [review]
1. There's a weird title faintly above the character's head.
image
2. In situations where there are several characters, a title is mounted on one character, and if you log on as a character without the title, the title effect that is being mounted on another character will appear.
image
image

**Review by Namyujeong — APPROVED** [review]
All problems have been solved!

image

image

File: nekoyume/Assets/_Scripts/Game/Game.cs [hunk 2]
```diff
@@ -135,13 +135,16 @@ private IEnumerator Start()
...
+            yield return StartCoroutine(ResourcesHelper.CoInitialize());
             AudioController.instance.Initialize();
             yield return null;
             // Agent 초기화.
             // Agent를 초기화하기 전에 반드시 Table과 TableSheets를 초기화 함.
             // Agent가 Table과 TableSheets에 약한 의존성을 갖고 있음.(Deserialize 단계 때문)
             var agentInitialized = false;
             var agentInitializeSucceed = false;
+
+            // yield return StartCoroutine();
```
Comment (boscohyun) [inline nekoyume/Assets/_Scripts/Game/Game.cs:147]:
```suggestion
```

**Review by boscohyun — APPROVED** [review]

**Pull request merged**